float64 amount
float64 vel
float64 forward_vel
---
float64 end_yaw
---
float64 start
float64 current
float64 target